e2e-tests/desktop/playwright.config.js

```diff
@@ -6,6 +6,18 @@ import { defineConfig } from '@playwright/test';
 
 /** @type {import('@playwright/test').PlaywrightTestConfig} */
 export default defineConfig({
+  globalSetup: '../global-setup',
   outputDir: './artifacts',
   workers: 1, // Tests need to be run in serial, otherwise there may be conflicts when using the CLI
+  use: {
+    baseURL: process.env.BOUNDARY_ADDR,
+    storageState: './admin/artifacts/authenticated-state.json',
```

Comment:
🤔  do we use the same auth state for both desktop and admin? In that should we delete the artifacts folder within ec2/desktop?

Reply:
I was thinking of doing a shared artifacts folder but we use them _slightly_ different right now so haven't pursued it yet, but we could do that.

Reply:
Agreed on the shared artifacts folder being a separate effort.